common/djangoapps/lang_pref/tests/test_api.py

```diff
@@ -10,3 +12,11 @@ class LanguageApiTest(TestCase):
     def test_released_languages(self):
         released_languages = language_api.released_languages()
         self.assertGreaterEqual(len(released_languages), 1)
+
+    @override_settings(ALL_LANGUAGES=[[u"cs", u"Czech"], [u"nl", u"Dutch"]])
+    def test_all_languages(self):
+        with translation.override('fr'):
+            all_languages = language_api.all_languages()
+
+        self.assertEqual(2, len(all_languages))
+        self.assertLess(all_languages[0][1], all_languages[1][1])
```

Comment:
Is is possible to also verify that the name of the language was actually translated?